sidecar/config/config.go

```diff
@@ -66,48 +66,34 @@ type Endpoint struct {
 	Type string `yaml:"type"`
 }
 
-// Registry configuration
-type Registry struct {
```

Comment:
Despite our discussion yesterday, I think we should keep (and deprecate) the top-level `registry:` and `controller:` for backwards compatibility. I'm aware of at least two users who are using config files with the sidecar, which would be broken by this.